README.md

```diff
@@ -113,6 +113,10 @@ Parsing dependencies information
 ...
 ```
 
+#### Optional environment variables
+* `MAX_PULL_REQUESTS` : Limit of creating PullRequest (MergeRequest)
+  * default. no limit
+
```

Comment:
I think we should query to find the existing number of PRs rather using only the counter. This code will only check current execution and will ignore all the potential existing Prs